packages/yourloops/test/integration/pages/patient-dashboard/patient-dashboard-for-HCP.spec.tsx

```diff
@@ -147,17 +145,19 @@ describe('Patient dashboard for HCP', () => {
     await act(async () => {
       renderPage(monitoredPatientDashboardRoute)
     })
-    const patientInfoCard = within(await screen.findByTestId('patient-info-card'))
-    const secondaryHeader = within(screen.getByTestId('patient-data-subnav-outer'))
+    const secondaryHeader = await screen.findByTestId('patient-nav-bar')
+    expect(secondaryHeader).toHaveTextContent('PatientMonitored PatientDate of birth:01/01/1980Diabete type:Type 1Gender:MaleRemote monitoring:YesShow moreDashboardDailyTrendsGenerate report')
 
-    expect(patientInfoCard.getByText(monitoredPatient.profile.fullName)).toBeVisible()
-    fireEvent.mouseDown(secondaryHeader.getByText(monitoredPatient.profile.fullName))
+    fireEvent.mouseDown(within(secondaryHeader).getByText(monitoredPatient.profile.fullName))
     fireEvent.click(screen.getByText(pendingPatient.profile.fullName))
 
-    await waitFor(() => {
-      // call this to update the card and catch the new patient
-      const patientInfoCard = within(screen.getByTestId('patient-info-card'))
-      expect(patientInfoCard.getByText(pendingPatient.profile.fullName)).toBeVisible()
-    })
+    const secondarHeaderRefreshed = await screen.findByTestId('patient-nav-bar')
```

Comment:
missing a letter but nice variable name 😉
```suggestion
    const secondaryHeaderRefreshed = await screen.findByTestId('patient-nav-bar')
```